src/main/java/com/miraisolutions/xlconnect/utils/RPOSIXDateTimeFormatter.java

```diff
@@ -179,19 +181,27 @@
             builder.appendLiteral(format.substring(i,i+1));
           }
         }
-        org.joda.time.format.DateTimeFormatter formatter = builder.toFormatter();
+        java.time.format.DateTimeFormatter formatter = builder.toFormatter();
         cache.put(format, formatter);
         return formatter;
     }
 
     public String format(Date d, String format) {
         StringBuffer sb = new StringBuffer();
-        getFormatter(format).printTo(sb, new org.joda.time.DateTime(d));
+        getFormatter(format).formatTo(ZonedDateTime.ofInstant(d.toInstant(), ZoneId.systemDefault()), sb);
         return sb.toString();
     }
 
     public Date parse(String s, String format) {
-        return getFormatter(format).parseDateTime(s).toDate();
+        java.time.format.DateTimeFormatter formatter = getFormatter(format);
```

Comment:
Is this according to the logic Joda's parseDateTime used? java.util.Date doesn't have any timezone information attached to it AFAIK.

Reply:
@martinstuder I changed the code to use LocalDateTime. It still requires a default zone offset / id to convert to Date via Instant.

Reply:
update: I tested that the (correct) behavior described in the [comments](https://github.com/miraisolutions/xlconnect/issues/50#issuecomment-460316521) of issue 50 remains unchanged.